### ISSUE TITLE
Do you plan to release the training code?

### ISSUE THREAD

**Issue by juxuan27** [kbrodt/sketch2pose#1]
Hi @kbrodt ! Thank you for your great work!
I'm really interested in this topic and can't wait to explore more about it. So I'm wondering will you release the training part of your model recently?

**Comment by kbrodt** [comment]
Hi,

The most training parts use different frameworks. For now you may:

- Train 2D pose estimation model via [Deep HRNet](https://github.com/leoxiaobin/deep-high-resolution-net.pytorch/) (or [mmpose](https://github.com/open-mmlab/mmpose)). The only part to change is the dataset format
- 1. Genere pseudo-labels of SMPL parameters via [SMPLify-X](https://github.com/vchoutas/smplify-x) (or [mmhuman3d](https://github.com/juxuan27/mmhuman3d)). Here you need to convert the keypoints to openpose [BODY_25](https://cmu-perceptual-computing-lab.github.io/openpose/web/html/doc/md_doc_02_output.html) format suitable for SMPLify-X.
  2. Finaly, train a [SPIN](https://github.com/nkolot/SPIN) network (or [mmhuman3d](https://github.com/juxuan27/mmhuman3d)). Here you also need to prepare the dataset and tweak the number of keypoints.

**Comment by juxuan27** [comment]
Thank you for your guidance! I'll try this pipeline later!
